aci/resource_aci_dhcpoptionpol.go

```diff
@@ -99,26 +94,25 @@ func getRemoteDHCPOptionPolicy(client *client.Client, dn string) (*models.DHCPOp
 	if err != nil {
 		return nil, err
 	}
-
 	dhcpOptionPol := models.DHCPOptionPolicyFromContainer(dhcpOptionPolCont)
-
 	if dhcpOptionPol.DistinguishedName == "" {
 		return nil, fmt.Errorf("DHCPOptionPolicy %s not found", dhcpOptionPol.DistinguishedName)
 	}
-
 	return dhcpOptionPol, nil
 }
 
-func getRemoteDHCPOptionFromDHCPOptionPolicy(client *client.Client, dn string) (*models.DHCPOption, error) {
-	dhcpOptionCont, err := client.Get(dn)
+func getRemoteDHCPOptionFromDHCPOptionPolicy(client *client.Client, dn string) ([]*models.DHCPOption, error) {
+	dhcpOptionCont, err := client.GetViaURL(fmt.Sprintf("%s/%s/%s.json", "/api/node/class", dn, "dhcpOption"))
 	if err != nil {
 		return nil, err
 	}
 
-	dhcpOption := models.DHCPOptionFromContainer(dhcpOptionCont)
+	dhcpOption := models.DHCPOptionListFromContainer(dhcpOptionCont)
 
-	if dhcpOption.DistinguishedName == "" {
-		return nil, fmt.Errorf("DHCPOption %s not found", dhcpOption.DistinguishedName)
+	for _, option := range dhcpOption {
+		if option.DistinguishedName == "" {
+			return nil, fmt.Errorf("DHCPOption %s not found", option.DistinguishedName)
```

Comment:
This error message tries to insert an empty string in the log

Reply:
Yeah, I'll change it.